cypress/integration/userCanLogout.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('User Log-Out', () => {
+   beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json"
+    })
+  });
+
+  it("Can log out successfully", () => {
+    cy.route({
+      method: 'DELETE',
+      url: 'http://localhost:3000/api/v0/auth/sign_out',
+      response: 'fixture:successful_user_logout.json',
+    })
+    cy.visit("http://localhost:3001")
+    cy.university_login("harvard@mail.com", "password");
+    cy.get("#logout-button").should("exist");
```

Comment:
There would be value in testing each type of account, not just university account.

Reply:
yes I thought about that too but wasnt sure. I'll continue adding that